dev/bots/analyze-sample-code.dart

```diff
@@ -212,50 +219,40 @@ dependencies:
       <String>['analyze', '--no-preamble', '--no-congratulate', mainDart.parent.path],
       workingDirectory: temp.path,
     );
-    stderr.addStream(process.stderr);
-    final List<String> errors = await process.stdout.transform<String>(utf8.decoder).transform<String>(const LineSplitter()).toList();
-    if (errors.first == 'Building flutter tool...')
+    final List<String> errors = <String>[];
+    errors.addAll(await process.stderr.transform<String>(utf8.decoder).transform<String>(const LineSplitter()).toList());
+    errors.addAll(await process.stdout.transform<String>(utf8.decoder).transform<String>(const LineSplitter()).toList());
+    if (errors.isNotEmpty && (errors.first.contains(' issues found. (ran in ') || errors.last.contains(' issue found. (ran in '))) {
+      errors.removeAt(0); // the "23 issues found" message goes onto stderr, which is concatenated first
+      if (errors.isNotEmpty && errors.last == '')
+        errors.removeLast();
+    }
+    if (errors.isNotEmpty && errors.first == 'Building flutter tool...')
       errors.removeAt(0);
-    if (errors.first.startsWith('Running "flutter packages get" in '))
+    if (errors.isNotEmpty && errors.first.startsWith('Running "flutter packages get" in '))
```

Comment:
Might this not be the first entry in `errors`, e.g. if there were lines written to `stderr`?

Reply:
(same issue with 'Building flutter tool...')

Reply:
the issues found line is the only one that goes to stderr, the rest are stdout.
I've added a null to separate the two and verified that after removing the one line, we see the null.